lib/march_hare/session.rb

```diff
@@ -82,12 +83,11 @@ def self.connect(options = {})
       case tls
       when true then
         cf.use_ssl_protocol
```

Comment:
We should kill this, this is also unsafe:

https://github.com/rabbitmq/rabbitmq-java-client/blob/5e0414a87778969aa919172685e0095343980198/src/main/java/com/rabbitmq/client/ConnectionFactory.java#L595

We should use a decent trust manager instead

Reply:
I agree but we only can do it when more options are provided.